src/Build/Evaluation/Evaluator.cs

```diff
@@ -2640,6 +2644,14 @@ private void SetAllProjectsProperty()
                     mayBeReserved: false);
             }
         }
+
+        private void VerifyVSDistributionPath(string path, ElementLocation importLocationInProject)
```

Comment:
Should this be conditional on being in .NET Core MSBuild? In Framework MSBuild if an import like this fails it's likely a missing VS workload or extension--which I guess we could also have a nice error for, too . . .